typescript/sdk/src/ccip/HyperlaneCCIPDeployer.ts

```diff
@@ -79,6 +90,32 @@ export class HyperlaneCCIPDeployer extends HyperlaneDeployer<
     return {};
   }
 
+  private async checkCCIPLanesSupport(
+    origin: ChainName,
+    config: Set<ChainName>,
+  ) {
+    this.logger.info(
+      `Checking CCIP lanes exist from ${origin} to ${Array.from(config).join(
+        ', ',
+      )}`,
+    );
+
+    const unsupportedLanes: string[] = [];
+    await Promise.all(
+      Array.from(config).map(async (destination) => {
+        const isSupported = await isSupportedCCIPLane({
+          origin,
+          destination,
+          multiProvider: this.multiProvider,
+        });
+        if (!isSupported) {
+          unsupportedLanes.push(destination);
+        }
+      }),
+    );
+    return unsupportedLanes;
```

Comment:
nit: use `.filter`

Reply:
ah yeah I was logging in here originally but can do a simple filter now